app/src/app/components/contents/admin/AdminLayout.tsx

```diff
@@ -21,10 +21,16 @@ export const AdminLayout = () => {
   }
 
   if (hasGlobalPacketManagePermission(authorities)) {
-    sidebarItems.push({
-      to: "/resync-packets",
-      title: "Resync Packets"
-    });
+    sidebarItems.push(
```

Comment:
this pushing onto a array multiple times may be better extracted into own function.. `getAdminLayoutSidebarItems`